### DISCUSSION TITLE
I need help to build

### DISCUSSION THREAD

**Discussion by Gokhanix** [tradingview/lightweight-charts#820] [General]
I have never developed a typescript project before.
So I'm having trouble building the project. 
Can someone give me a step-by-step to compile/build it?

--
Methods I've tried
Downloaded the project to a folder.
Opened it with visual studio code.
Entered the 'npm run tsc' code from the terminal.
The answer I got:

> lightweight-charts@3.5.0 tsc
> ttsc -p tsconfig.prod.json
>'ttsc' is not recognized as an internal or external command,
operable program or batch file.

**Comment by timocov** [comment]
The first, you have to install all dependencies - `npm install`. After that you can use `npm run build` script to build the project https://github.com/tradingview/lightweight-charts/blob/master/BUILDING.md.

**Comment by Gokhanix** [comment]
![rr1](https://user-images.githubusercontent.com/89110881/129882183-fa73ae3f-11bc-429c-9eea-8a9621255b56.png)
Im enter the command but got error

**Reply by timocov** [reply]
Please use npm@6

**Reply by mariamlouis** [reply]
I face the same problem npm is not working Please help
